src/components/mods/modsTable.tsx

```diff
@@ -31,9 +32,14 @@ const useStyles = createStyles(
     ) => {
         return ({
             tabContainer: {
-                padding: "0 15px",
+                height: "29px",
```

Comment:
Does this need to be here? I couldn't figure out what it was doing in my testing, but I didn't want to just remove it without asking.

Reply:
Yeah the `height` isn't required, I've removed it.